src/features/snap/components/ReportSnap.test.tsx

```diff
@@ -26,7 +26,7 @@ describe('ReportSnap', () => {
     const { queryByText } = render(
       <ReportSnap
         snapName="Snap1"
-        snapId="Snap1ID"
+        versionChecksum="mockv1checksum"
```

Comment:
there is a constant from test-utils, please use SNAP_SHASUM_1

Reply:
will apply.